bot-samples/instagram-template/README.md

```diff
@@ -0,0 +1,71 @@
+# Instagram Starter Guide
+This guide includes:
+- A bot template for Instagram which can detect which "entry point" the consumer is accessing the bot through, and route to a corresponding Dialog
+- A guide to integrating LivePerson with Instagram (accurate as of Oct 2021)
+- A video of a practical example in action
+
+The template should help you get up and running quickly with LivePerson's Instagram integration.
+
+### Prerequisites
+
+- Access to the Conversational Cloud and a basic knowledge of LivePerson bots
+- A Facebook Business Page [connected to an Instagram account](https://www.facebook.com/help/1148909221857370)
+- The Instagram account should [allow access to direct messages](https://www.facebook.com/help/instagram/791161338412168)
+- An Instagram-specific Skill, Agent, and active Agent Connector, [connected to the uploaded bot](https://developers.liveperson.com/tutorials-guides-getting-started-with-bot-building-deploy-the-bot.html) associated with this guide
+- Correct configuration of Conversational Cloud account settings in Houston (ask your account representative if you're not a LivePerson employee)
+    - AC Features:
```

Comment:
I'm not sure if this stuff is OK to be publicly exposed. I know it would help internal folks, but maybe we can't show these things seeing as it'll be externally available.

Reply:
I think that we should be listing out these features where appropriate. For those that do not have access to Houston, they're meaningless, and for those that do, it's extremely beneficial. As long as we explain that if you do not have access to the platform that you need to contain your LP rep, we should be good to go.